modules/services/conky.nix

```
@@ -0,0 +1,55 @@
+{ config, lib, pkgs, ... }:
+
+let cfg = config.services.conky;
+
+in with lib; {
+
+  meta.maintainers = [ maintainers.kaleo ];
+
+  options = {
+    services.conky = {
+      enable = mkEnableOption "Light-weight system monitor";
```

The description for `mkEnableOption` is prefixed by `Whether to enable ...`, so something like this would be more appropriate:

```suggestion
      enable = mkEnableOption "conky, a light-weight system monitor";
```